db/migrate/20230606150010_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :author_id
+      t.integer :post_id
```

Comment:
- [ ] Kindly make the `author_id` and `post_id` columns have a `null: false` constraint. This would ensure that the columns won't have a `nil` value when an instance of the `Comment` model is created. 👍 😎